.github/workflows/lint.yaml

```diff
@@ -156,6 +156,10 @@ jobs:
         # We have to add the current directory as a safe directory or else git commands will not work as expected.
         run: git config --global --add safe.directory $(realpath .) && make derive-up-to-date
 
+      - name: Check if go generated files are up to date
+        # We have to add the current directory as a safe directory or else git commands will not work as expected.
+        run: git config --global --add safe.directory $(realpath .) && make go-generate-up-to-date
```

Comment:
I don't think this needs the `git config` command repeated from the last step - it only needs to be done once. I'm pretty sure it's not even needed on the previous command as `actions/checkout@v4` defaults to setting the checkout directory as a safe directory: https://github.com/actions/checkout/blob/main/action.yml#L95